docs/superpowers/plans/2026-04-17-trigger-policy.md

```
@@ -0,0 +1,854 @@
+
+        return None
+
+    if trigger_mode == TriggerMode.ANYWHERE:
+        result = _try_direct_match(body, default_translation)
+        if result:
+            return TriggerMatch(passage=result[0], translation=result[1], source=TriggerSource.DIRECT)
+
+        result = _try_embedded_match(body, default_translation)
+        if result:
+            return TriggerMatch(passage=result[0], translation=result[1], source=TriggerSource.ANYWHERE)
+
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)